### PR TITLE
Topnav

### PR DESCRIPTION
**Pull request by HorstFrank**
One of my Pages has a Top Navigation ... so this is the Component

### CODE REVIEW

File: pages/settings.tsx [hunk 1]
```diff
@@ -2,7 +2,15 @@ import React from "react";
...
 import Header from "../components/header/Header";
 import styles from "../styles/Home.module.css";
+import ReactPlayer from "react-player/youtube";
 
 export default function Home() {
-  return <div className={styles.container}>settings view</div>;
+  return (
+    <div className={styles.container}>
+      <ReactPlayer
+        url="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```
Comment (fre-ben) [inline pages/settings.tsx:11]: Fantastic! 🎤

**Review by fre-ben — APPROVED** [review]
LGTM, maybe you could delete some commented out code when making a PR :)
Also a deployment would be nice so we could see what you're doing :)
(liked the song)

**Review by RonjaMeinecke — APPROVED** [review]
💥good job

File: assets/json/text.json [hunk 2]
```diff
@@ -4,5 +4,63 @@
...
     "settings": "Einstellungen"
+  },
+  "topnav": {
+    "thisapp": "Diese App",
+    "privacy": "Datenschutz",
+    "imprint": "Impressum"
+  },
+  "infotext": {
+    "thisapp": "Dies ist der Text zu diese App",
+    "privacy": "Dies ist der text zum Datenschutz",
```
Comment (SilviaBarcellini) [inline assets/json/text.json]:
```suggestion
    "privacy": "Dies ist der Text zum Datenschutz",
```

File: assets/json/text.json [hunk 3]
```diff
@@ -4,5 +4,63 @@
...
+    "imprint": "Dies ist der text zum Impressum"
```
Comment (SilviaBarcellini) [inline assets/json/text.json:16]:
```suggestion
    "imprint": "Dies ist der Text zum Impressum"
```

File: assets/json/text.json [hunk 4]
```diff
@@ -4,5 +4,63 @@
...
+    "welcome": {
+      "label": "Willkommen",
+      "content": "Du willst sicher gleich loslegen mit den Fragen. Doch vorher solltest du einiges über diese App wissen."
+    },
+    "thisapp": {
+      "label": "Warum ist diese App schlecht?",
+      "content": "Das Prinzip dieser App führt dazu, dass nur die etablierten Parteien berücksichtigt werden."
+    },
+    "good": {
+      "label": "Warum ist dies App gut?",
```
Comment (SilviaBarcellini) [inline assets/json/text.json]:
```suggestion
      "label": "Warum ist diese App gut?",
```

**Review by SilviaBarcellini — APPROVED** [review]
Just spotted a couple of typos ✏️ 
the rest lgtm 👍

File: pages/settings.tsx (hunk 1 above)
Comment (HorstFrank) [reply pages/settings.tsx:11]: thanks! yes ... deployment a weekend project, maybe this WE.

**Pull request merged**